packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -348,8 +348,20 @@ const UnifiedDesignPicker: React.FC< UnifiedDesignPickerProps > = ( {
 } ) => {
 	const hasCategories = !! Object.keys( categorization?.categories || {} ).length;
 
-	// Track as if user has scrolled to bottom of the design picker
-	const ref = useInView< HTMLDivElement >( onViewAllDesigns, {}, [ categorization?.selection ] );
+	const [ viewedSelections, setViewedSelections ] = useState< string[] >( [] );
+
+	const { ref } = useInView( {
+		onChange: ( inView ) => {
+			if (
+				inView &&
+				categorization?.selection &&
+				! viewedSelections.includes( categorization?.selection )
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg) ![badge](https://img.shields.io/badge/review-imo-orange.svg)

This condition will likely prevent the event from firing if the user scrolls to the end a second time in the same category.

e.g., 

- Select `Blog`
- Scroll to the bottom 
	- The event fire 🔥 
- Scroll to the top and Select `Business`
- Scroll to the bottom 
	- The event fire 🔥 
- **Scroll to the top and Select `Blog` again**
- Scroll to the bottom 
	- **The event does not fire** 🧑‍🚒 

I thought this was not intended and should be changed to the original behavior, but WDYT? @Automattic/lego @autumnfjeld

Reply:
> This condition will likely prevent the event from firing if the user scrolls to the end a second time in the same category.

This is correct. I thought this was the original intent...

Reply:
@danielbachhuber thank you for working on this 🙂
I went to the production environment to double-check, and confirmed that the event would trigger again after switching categories (which means that there is no need to keep track of the viewed categories).